frontend/src/app/universities/universities.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { UniversitySearchService} from '../services/university-search.service';
+import { University} from '../models/university';
+
+@Component({
+    selector: 'app-universities',
+    templateUrl: './universities.component.html',
+    styleUrls: ['./universities.component.css'],
+    providers: [UniversitySearchService]
+})
+
+export class UniversitiesComponent implements OnInit {
+
+    universities: University[];
+    fatalMessage: string = null;
+    sysErrorMessageTemplate: string = "Universities system has encountered an error, please refresh.";
+    message: string = null;
+
+    constructor(private universitySearchService: UniversitySearchService) {
+
+    }
+
+    ngOnInit() {
+        this.fetch();
+    }
+
+    cleanMessages(){
+        this.fatalMessage = null;
+        this.message = null;
+    }
+
+    fetch(){
+        this.cleanMessages();
+        this.universitySearchService.fetch()
+            .then(result => {
+                if(result.isSuccessful()){
+                    this.universities = result.response;
+                } else {
+                    this.fatalMessage = result.response['message'];
```

Comment:
Access the error message using the field result.response.message.

If we ever renamed the variable inside SubjectHubApiResponse, the IDE might not pick this up.